{
  "schema_version": "1.3.0",
  "id": "GHSA-v3c5-jqr6-7qm8",
  "modified": "2022-12-23T00:30:23Z",
  "published": "2022-12-23T00:30:23Z",
  "aliases": [
    "CVE-2022-40899"
  ],
  "details": "An issue discovered in Python Charmers Future 0.18.2 and earlier allows remote attackers to cause a denial of service via crafted Set-Cookie header from malicious web server.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-40899"
    },
    {
      "type": "WEB",
      "url": "https://github.com/python/cpython/pull/17157"
    },
    {
      "type": "WEB",
      "url": "https://github.com/PythonCharmers/python-future/blob/master/src/future/backports/http/cookiejar.py#L215"
    },
    {
      "type": "WEB",
      "url": "https://pypi.org/project/future/"
    },
    {
      "type": "WEB",
      "url": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": null,
    "github_reviewed": false
  }
}